examples/src/examples/graphics/lit-material.tsx

```diff
@@ -98,27 +99,20 @@ class CustomMaterialExample {
                 light.translate(0, 1, 0);
                 app.root.addChild(light);
 
-                const material = new pc.CustomMaterial();
+                const material = new pc.LitMaterial();
                 material.setParameter("texture_envAtlas", assets.helipad.resource);
                 material.setParameter("material_reflectivity", 1.0);
                 material.setParameter("material_normalMapIntensity", 1.0);
                 material.setParameter("texture_diffuseMap", assets.color.resource);
                 material.setParameter("texture_glossMap", assets.gloss.resource);
                 material.setParameter("texture_normalMap", assets.normal.resource);
-                const options = new pc.LitOptions();
-                options.shadingModel = pc.SPECULAR_BLINN;
-                options.useSpecular = true;
-                options.useMetalness = true;
-                options.occludeSpecular = 1;
-                options.reflectionSource = 'envAtlas';
-                options.reflectionEncoding = assets.helipad.resource.encoding;
-                options.ambientSource = 'envAtlas';
-                options.ambientEncoding = assets.helipad.resource.encoding;
-                options.clusteredLightingEnabled = app.scene.clusteredLightingEnabled;
-                options.normalMapEnabled = true;
-                options.useSpecularColor = true;
-                options.useSpecularityFactor = true;
-                material.litOptions = options;
+                material.shadingModel = pc.SPECULAR_BLINN;
+                material.useSkybox = true;
+                material.hasSpecular = true;
+                material.hasSpecularityFactor = true;
+                material.hasNormals = true;
+                material.hasMetalness = true;
+                material.occludeSpecular = 1;
```

Comment:
Should you maybe use the constant here? i.e. `SPECOCC_AO`

Reply:
Yes! Done! Thanks!